src/python/TaskWorker/Main.py

```diff
@@ -1,23 +1,21 @@
-from optparse import OptionParser
+"""
+Top driver for TaskWorker service
+"""
+from optparse import OptionParser  # pylint: disable=deprecated-module
 import signal
 import os
 import logging
 import sys
 
 from WMCore.Configuration import loadConfigurationFile
 from TaskWorker.WorkerExceptions import ConfigException
```

Comment:
Shall we also move this line to late binding? Just in case.
And maybe put a comment here that you need to do late import for import crab relate code.

Reply:
My apologies. I do not understand which line is `this line` and where you suggest to move it to. OK for the comment.

Reply:
Ah...I mean line 11. 
```python
 from TaskWorker.WorkerExceptions import ConfigException
```
I thought when I click at line 11 (which does not contain any diff) it will only show line 11.

Reply:
got it. Yes. Thanks

Reply:
Done